### DISCUSSION TITLE
airflow scheduler did not trigger the weekly cron dag

### DISCUSSION THREAD

**Discussion by TomMonkeyMan** [apache/airflow#25468] [General]
### Apache Airflow version

2.0.1

### What happened

hi, I'm planning the dag with bash operator to be run every Monday 8 am with "00 08 * * 1", but the dag did not get triggered. tested with "00 08 * * 1-2" and it was working well. can anyone help with it?

### Operating System

Debian GNU/Linux 10

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Please don't use Issues for troubleshooting requests

**Reply by potiuk** [reply]
Look at the way airlfow schedules things at the end interval not at the beginning by default. There is a new #23662 that might be there in 2.4 that might give you a chance for the scheduler to behave more cron-like

**Comment by freeyoung** [comment]
Maybe your start_date needs to be modified to be a static date that is at least one week earlier than today
